foo.js

```diff
@@ -0,0 +1,3 @@
+
+debugger;
+const foo = 1;
```

Comment:
I should've added a .gitgnore on the root directory

Reply:
oops, my bad 😅  I removed it